### ISSUE TITLE
Copyright, attribution, etc questions for my fork

### ISSUE THREAD

**Issue by briandominick** [vsoch/docsy-jekyll#47]
Hi @vsoch , I am posting here to let you know that I have forked this awesome theme and am maintaining an unsynced repo. I have maintained heavy attribution and all your copyright notices and so forth, but that this point I think there is little left of your original code, and most of it is in the general layouts and templates. I expected to hue much closer to what you had, because I really liked it, but at this point I'm more concerned that your name is all over something that maybe doesn't look much like your work.

I'm fine with carrying all that attribution forward, but I'd slightly prefer to reduce it to a mention somewhere. For instance, I'm happy to credit/link to you in the first paragraph of the Readme in perpetuity. So if you have a chance to look and you'd rather have your name removed from files and the copyright file, or whatever you prefer, please let me know here. (I could issue a PR to this repo if that will help you compare, or you can do it on your end.)

https://github.com/DocOps/asciidocsy-jekyll-theme

Thanks!

**Comment by vsoch** [comment]
@briandominick for any open source work you can’t just fork it and make some claim that it’s different enough to remove the original author - if you did want to have my (or any author for that matter) attribution you should not have started with someone else’s work. And I disagree that your template is widely different - it looks like a modified version of the template here.  Regardless, the right thing to do is share the attribution for the work, here is an example:

https://opensource.stackexchange.com/questions/5528/removing-copyright-notice-in-uis-of-apache-2-licensed-software

**Comment by briandominick** [comment]
Whoa whoa whoa, I think you took me all wrong. 

> for any open source work you can’t just fork it and make some claim that it’s different enough to remove the original author 

Like, I literally posted here and ASKED you, so I didn't "make some claim that it's different enough". That's what this thread is about, right? Why respond like that's what I did, when in fact you're credited ALL OVER my repo and I wrote to explicitly ask with an open-ended array of options.

My templates are indeed radically different from yours -- I know from the hundreds of hours of work I put into them, but credit was not my point -- I was asking if you still wanted your name on files that had been changed so much, and if you still wanted the SOLE copyright in the COPYRIGHT notice that I'm not allowed to remove. This was a courtesy to avoid confusion -- I am NOT trying to steal credit for your work. I do not give even the slightest crap about credit and have never responded to outright theft without attribution as harshly as you just responded to me asking politely if you would prefer I change it. I praised you here, I credited and praised you in my Readme, I'm a big fan of your work, I pay explicit homage to it, and I cannot fathom your response to me.

But anyway, I take your answer as a no. Fair enough.

**Comment by briandominick** [comment]
I see that I can append my name to the Copyright according to your link, so I'll do that. I'm very sorry that this upset you. I never intended or tried to undermine your copyright, I assure you.

**Comment by vsoch** [comment]
I can try to explain how I read your issue, if it helps!

> but that this point I think there is little left of your original code, and most of it is in the general layouts and templates. I expected to hue much closer to what you had, because I really liked it, but at this point I'm more concerned that your name is all over something that maybe doesn't look much like your work.

This to me reads as saying "this doesn't look like your work anymore and I'm _concerned_ to have your name on it."  Are you concerned for me? Why would I be concerned to get attribution for something that was derived from a work I'm proud of? It gives me joy to see that you made a variation, and it links back to the original work.

>  I'd slightly prefer to reduce it to a mention somewhere.

And I read this as a nice way of asking to remove it, or put it somewhere that is less prominent. I apologize if I misunderstood, but the above together does appear like you are asking (nicely) to remove or minimize my attribution.  It's a bit of an awkward question.

To answer your question, I think the right thing to do is what you are already doing, pointing out any previous attributions for your derived works so there is no question or doubt. And you have extended the work, so you definitely should put your name alongside the Copyright to take credit for that. But being careful about attribution is something I would encourage - it fosters good feelings between project maintainers, and (at least for me) makes me feel honest and upfront about my work. This is [a real issue](https://link.springer.com/chapter/10.1007/978-3-642-39891-9_4) and (speaking for myself) it feels good to be on the right side of it.

I'm glad that you've found the project useful and have turned it into a new derivation, it makes me happy to see that! I think we've found a good solution so please close the issue if you are also content.

**Comment by vsoch** [comment]
And thank you for your apology, it's eons above so many interactions I've had on GitHub issues! I really appreciate it.

**Comment by briandominick** [comment]
I'm sorry you're had bad experiences with this in the past. I can see how that might make you expect the worst. I promise I was coming from the opposite perspective, because I would very very very much not want my me on anything that I did not largely offer or have a say in. So I really was asking if you still wanted all this weird stuff I'm doing associated with your good name lol. I have no problem honoring the open source license but obviously people should be able to beg out of attribution. I spent the last few months worried go might get in touch with me and ask for your name removed lol. We probably come from opposite perspectives. My name is all over stuff I have nothing to do with and at points it has made my life pretty hard. Glad we figured this out. Proud to keep your name on my (our) codebase.
